requirements.txt

```diff
@@ -0,0 +1 @@
+django
```

Comment:
You need to specify a version, otherwise the latest version will be always installed and that could break the build of the app.